From 58546ce76063d7cc41a6601b44eca9d5502d9da1 Mon Sep 17 00:00:00 2001
From: Dridi Boukelmoune <dridi.boukelmoune@gmail.com>
Date: Mon, 4 Mar 2024 23:16:27 +0100
Subject: [PATCH 15/32] common: Increase node string buffer size

Since the maximum number of CPUs was doubled, it might be reasonable to
double the size of the buffer for the string representation.

Refs 6f6cc3b24d84c413556639b64a62aca6ad0b21cc
---
 common/os/os_win.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/os/os_win.c b/common/os/os_win.c
index 69e24be..bd6f5ce 100644
--- a/common/os/os_win.c
+++ b/common/os/os_win.c
@@ -215,7 +215,7 @@ nodedetail_line_show(win_reg_t *reg, char *title, char *value, int line)
 void
 os_nodedetail_data(dyn_nodedetail_t *dyn, win_reg_t *seg)
 {
-	char s1[256], s2[32];
+	char s1[512], s2[32];
 	node_t *node;
 	win_countvalue_t value;
 	node_meminfo_t meminfo;
-- 
2.41.0

